docs/source/kale.embed.rst

```diff
@@ -14,6 +14,16 @@ kale.embed.attention\_cnn module
    :undoc-members:
    :show-inheritance:
 
+kale.embed.deepdta module
```

Comment:
deep_dta?

Reply:
Sorry forgot to verify docs, new commit submitted and deep_dta is correct.